### ISSUE TITLE
Update alt text for Zero Hunger image on the About Us page to adhere to WCAG

### ISSUE THREAD

**Issue by kathrynsilvaconway** [hackforla/website#3382]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). In this issue, we will change the alt text for the Radical Inclusivity image on the [About Us page](https://www.hackforla.org/about/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] In the file `_includes/about-page/about-card-sustainability.html`, change the following alt text

From:
```
Zero Hunger goal logo
```

To:
```
2 Zero Hunger
```

- [ ] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3361 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3361 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3361 
- [About Us page](https://www.hackforla.org/about/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by kyrakwak** [comment]
Availability for this week: Weekdays flexible
ETA: 1/17 (today) 9AM
